geode-core/src/main/java/org/apache/geode/internal/net/NioSslEngine.java

```diff
@@ -308,18 +308,17 @@ void expandPeerAppData(ByteBuffer wrappedBuffer) {
     }
   }
 
-  /*
-   * NioSslEngine doesn't need to ensure capacity in network buffers because they
-   * are fixed in size by the SslContext. The size recommended by the context is
-   * big enough for the SslEngine to do its work.
-   */
   @Override
   public ByteBuffer ensureWrappedCapacity(int amount, ByteBuffer wrappedBuffer,
```

Comment:
The `amount` param is no longer used, but I'm not sure if you are bound to this method signature and therefore cannot remove it (looks like you are overriding a parent class' method).

Reply:
The signature of this method is from the interface NioFilter and `amount` is used by another implementation of that interface.